plugins/module_utils/module_index.py

```diff
@@ -176,4 +176,22 @@
             },
         },
     },
```

Comment:
Is that a version 24.1 feature only? Otherwise I would add the paths for the versions 22.7, 23.1, 23.7 as stated in  molecule/system_settings_logging/molecule.yml

Reply:
good point, `max_log_file_size_mb` is only available from 24.1 onwards, the other option is available since at least 22.7. I've updated the `VERSION_MAP` accordingly. But the converge tests will fail since they are identical for all version. I will find a solution in the upcoming days

Reply:
this should be fixed now